pkg/openstack/actuator.go

```diff
@@ -213,6 +229,11 @@ func (a *OpenStackActuator) GetCredentialsRootSecretLocation() types.NamespacedN
 	return types.NamespacedName{Namespace: constants.CloudCredSecretNamespace, Name: constants.OpenStackCloudCredsSecretName}
 }
 
+// GetCredentialsRootSecretLocation returns the namespace and name where the CACert is stored.
+func (a *OpenStackActuator) GetCACertConfigMapLocation() types.NamespacedName {
+	return types.NamespacedName{Namespace: constants.OpenShiftConfigNamespace, Name: constants.CloudProvderConfigMapName}
+}
```

Comment:
This PR is just copying the CA cert, which we know must exist at this location, into every credentials secret? Is the goal to make it a little easier for someone to use the cred secret and not have to do a second get?

Reply:
It requires the operator to have additional permissions to get stuff from openshift-config namespace that it does not need to have.